CMakeLists.txt

```diff
@@ -92,11 +105,11 @@ if(NOT DAEMON_EXTERNAL_APP)
     elseif(WIN32)
         set(USE_CURSES ON)
     else()
-	option(USE_CURSES "Compile with curses support" 1)
+        option(USE_CURSES "Enable fancy colors in terminal's output (does not detect if output is a file)" ON)
     endif()
     cmake_dependent_option(USE_SMP "Compile with support for running the renderer in a separate thread" 1 BUILD_CLIENT 0)
-    cmake_dependent_option(USE_GEOIP "Use libgeoip" 1 "BUILD_SERVER OR BUILD_CLIENT OR BUILD_TTY_CLIENT" 0)
-    option(USE_BREAKPAD "Use Breakpad crash logging" 0)
+    cmake_dependent_option(USE_GEOIP "Allows to show from which coutry a player logs from" 1 "BUILD_SERVER OR BUILD_CLIENT OR BUILD_TTY_CLIENT" 0)
```

Comment:
"Server prints which country a player connects from"

Reply:
this imply that this dependency is server-side only. I don't know if that's true or wrong.

Reply:
Yes it is server-side only. It takes the client IP and looks it up in the database